lib/ddtrace/contrib/elasticsearch/patcher.rb

```diff
@@ -41,22 +41,16 @@ def patch_elasticsearch_transport_client
 
             def initialize(*args, &block)
               service = Datadog::Tracing.configuration[:elasticsearch][:service_name]
+              Datadog::Tracing.configure_onto(self, service_name: service)
```

Comment:
Do we *need* to configure a pin in this `Elasticsearch::Transport::Client` initializer?

It seems like it only keeps track of `Datadog::Tracing.configuration[:elasticsearch][:service_name]`, which is a global configuration.

Reply:
No, I don't think we must, I think I was mostly recreating existing behavior here.